cookbooks/bcpc/recipes/graphite.rb

```diff
@@ -189,8 +194,11 @@
     action :nothing
     user "root"
     code <<-EOH
-        python /opt/graphite/webapp/graphite/manage.py syncdb --noinput
-        python /opt/graphite/webapp/graphite/manage.py createsuperuser --username=admin --email=#{node[:bcpc][:admin_email]} --noinput
+        export PYTHONPATH='/opt/graphite/webapp'
```

Comment:
The indentation in this file is not consistent; it has indentation of 2(which is standard for chef-bach) and 4. If its not too much work, it would be good to fix the indentation on this file. This way, whenever we modify a file, we can improve it.